.github/workflows/lint.yml

```
@@ -10,11 +10,12 @@ jobs:
     - name : master base sha
       id : master_base_sha
       if : github.ref  == 'refs/heads/master'
-      run: echo  "::set-output name=base_sha::${{ github.event.before }}"
+      run: echo "base_sha=${{ github.event.before }}" >> $GITHUB_OUTPUT
     - uses: xcoo/clj-lint-action@v1.1.11
       with:
         linters: "[\"clj-kondo\" \"kibit\" \"eastwood\"]"
         github_token: ${{ secrets.GITHUB_TOKEN }}
         runner: ":leiningen"
         base_sha: ${{ github.event.pull_request.base.sha||steps.master_base_sha.outputs.base_sha }}
         eastwood_linters: "[:all]"
+        linter_options: "{:eastwood {:exclude-linters [:keyword-typos :unused-namespaces]}}"
```

Let's use compiler warnings instead of eastwood

```suggestion
        linter_options: "{:eastwood {:exclude-linters [:keyword-typos :unused-namespaces :boxed-math]}}"
```

https://github.com/chrovis/cljam/blob/7eeaf02b9ef4167ac2104df01dd94364ffcfcb3b/.github/workflows/main.yml#L32-L35